src/common.rs

```diff
@@ -217,6 +220,19 @@ impl Config {
     }
 }
 
+#[cfg(feature = "tmp")]
+fn tempdir() -> PathBuf {
+    tempdir::TempDir::new("compiletest")
+        .expect("failed to create temporary directory")
+        .path()
+        .to_path_buf()
+}
```

Comment:
Won't the `TempDir` be dropped here, deleting the temporary directory? Will the directory be recreated elsewhere, or does the caller of this method assume a directory exists at the returned `PathBuf`? If the latter, then this function is buggy.

Reply:
You might be right. I'll look into it. The test project runs fine though...

Reply:
I think the test runner takes care of creating the dir (in essence TempDir just provides the name)

Reply:
Perhaps we should use [into_path](https://doc.rust-lang.org/tempdir/tempdir/struct.TempDir.html#method.into_path) instead? According to the documentation, `into_path` leaves the directory intact.

Reply:
@Munksgaard Thanks, that's exactly what I needed.